pgmpy/independencies/Independencies.py

```diff
@@ -49,6 +49,9 @@ def __str__(self):
         return string
 
     __repr__ = __str__
+    
+    def __eq__(self, other):
+        return self.__str__()==other.__str__()
```

Comment:
@kshitij10496 I will need to see the code to understand why wasn't it working. And since each `Independencies` class has an attribute `independencies` which is a list, you will need to sort this list somehow before comparing. And also since each element of this list would be an `IndependenceAssertion` instance, you will need to add `__eq__` method in this class also so that you can compare these objects.